content/2015.09.encrypter-les-donnees-en.rst

```diff
@@ -0,0 +1,101 @@
+End-to-end encryption using Kinto.js
+###########################################
+
+:date: 2015-09-14
+:summary: How to encrypt data before sending it to the server.
+:slug: encrypter-les-donnees
+:lang: en
+:url: encrypter-les-donnees
+
+A guest post this time, by Michiel de Jong from the Firefox OS team! :)
+
+As a first step in the Firefox OS
+"[Data Sync](https://wiki.mozilla.org/Firefox_OS_Data_Sync)" project, we're working on
```

Comment:
If you prefer to have it written in md, it's okay, but you need to name your article with `.md` at the end.